packages/xo-server/src/xapi/index.js

```diff
@@ -2298,9 +2302,7 @@ export default class Xapi extends XapiBase {
 
     // ignore errors, I (JFT) don't understand why they are emitted
     // because it works
-    await this._importVdiContent(vdi, buffer, VDI_FORMAT_RAW).catch(
-      console.warn
-    )
+    await this._importVdiContent(vdi, buffer, VDI_FORMAT_RAW).catch(log.warn)
```

Comment:
This is not a correct use of `log.warn`, maybe we should update the API to allow this though.

Reply:
Are u sure ? After test, log.warn works with juste `.catch(log.warn)`. It log the error.

Reply:
The first param of `log.warn` must be a string message.

Error objects should be passed in `data`.